### DISCUSSION TITLE
Mandatory two-factor authentication is evil and needs to be removed from GitHub

### DISCUSSION THREAD

**Discussion by jbruchon** [community/community#118703] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

**I deleted ALL of my repositories and moved them elsewhere over forced 2FA.**

[Leaving GitHub: forced two-factor authentication is evil](https://www.jdupes.com/2023/08/28/news-hash-database-feature-cross-copying-hard-link-hashes-jdupes-leaving-github/)

> The increased security offered by 2FA comes from requiring verification outside of the traditional “give me your password” system of logging in; someone who gets their hands on your password probably won’t also have your unlocked cell phone in their hand when they try to log into your account using that stolen password, and that’s how and why 2FA works in a nutshell. Unfortunately, this also means that your password isn’t enough for YOU to log in, either; if you lose access to your second factor, you lose your account even if you know your password, and that’s how and why 2FA is a bad thing in a nutshell

[Mandatory Two-Factor Authentication Is Bad And Stupid](https://www.youtube.com/watch?v=bbmsIwopGM8)

**Comment by leofun01** [comment]
Totally agree.
